docs/pages/application-access/getting-started.mdx

```diff
@@ -23,38 +36,42 @@ Let's connect to Grafana using Teleport in three steps:
 
 - (!docs/pages/includes/tctl.mdx!)
 - A host where you will run the Teleport Application Service.
+- A DNS subdomain for each application.
+   
+   <Admonition type="note" title="Application Access and DNS for Self-Hosted Clusters">
 
-<Tabs>
-<TabItem label="Docker">
-- A Docker installation, which we will use to launch Grafana in a container.
-  Alternatively, if you have another web application you'd like to protect with
-  Teleport, you can use that instead.
-</TabItem>
-<TabItem label="Kubernetes Cluster">
+   (!docs/pages/includes/dns-app-access.mdx!)
 
-(!docs/pages/includes/kubernetes-access/helm-k8s.mdx!)
+   Teleport Cloud will automatically provide a subdomain and signed TLS certificate for your application under your tenant address.
+   </Admonition>
 
-- Administrative rights to deploy within a cluster.
+- Docker or Kubernetes to launch Grafana.
 
-</TabItem>
-</Tabs>
+   This tutorial illustrates how to launch Grafana in a Docker container or Kubernetes cluster.
+   Therefore, you must have either Docker or Kubernetes installed. For more information, click **Docker** or **Kubernetes cluster**.
```

Comment:
I suggest removing this paragraph as unnecessary. Docker/Kubernetes is mentioned in the second sentence of the first paragraph of the page, again in the 2nd bullet point of the first list, and directly above and below. I'm not sure this adds anything that isn't already explained.